src/components/faceVerification/sdk/FaceTecCore.js

```diff
@@ -1,2 +1,19 @@
 // this is a hack for vite.
-export default { FaceTecSDK: global.FaceTecSDK } // FaceTecSDK is included as js script in index.html
+const injectScript = src => {
+  return new Promise((resolve, reject) => {
+    const script = document.createElement('script')
+    script.src = src
+    script.addEventListener('load', resolve)
+    script.addEventListener('error', e => reject(e.error))
+    document.head.appendChild(script)
+  })
+}
+
+let injected = false
+export const importSDK = async () => {
```

Comment:
could we avoid global let by wrapping it at loadsh once ?

Reply:
done